src/main/java/Storage.java

```diff
@@ -0,0 +1,123 @@
+
+    String FILEPATH; 
+
+    public Storage(String filepath) {
+        this.FILEPATH = filepath;
+    }
+
+    /**
+     * Count file rows.
+     * @param file file
+     * @return file row count
+     * @throws IOException
+     */
+    public long getLineCount(File file) throws IOException {
+        try (Stream<String> lines = Files.lines(file.toPath())) {
+            return lines.count();
+        }
+    }
+
+    public void increaseSize(String filepath) throws IOException {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(filepath)); 
+            FileWriter writer = new FileWriter(filepath);
+            
+    
+            for (int i = 0; i < 1; i++) {
+                int prev = reader.read();
+                int now = prev + 1;
+                writer.write(now);
+            } 
+            reader.close();
+            writer.close();
+        } finally {
+        
+        }
+    }
+
+    /**
+     * retrieve task from storage
+     * @param lineContent
+     * @param filepath
+     * @return String task
+     * @throws IOException
+     */
+    public String getTask(String lineContent, String filepath) throws IOException {
+        File file = new File(filepath);
+        return Files.lines(file.toPath())
+                    .filter(line -> line.contains(lineContent))
+                    .reduce("", (x,y) -> x+y+'\n');
+    }
+
+    public String getTaskFromNumber(String num, String filepath) throws IOException {
+        File file = new File(filepath);
+        return Files.lines(file.toPath())
+            .filter(line -> !line.substring(0,1).equals(num))
+            .reduce("", (x,y) -> x+y+'\n');
+    }
+
+    /**
+     * remove task from storage
+     * @param lineContent
+     * @param filepath
+     * @param temp
+     * @throws IOException
+     */
+    public void removeLine(String lineContent, String filepath, String temp) throws IOException {
+        File file = new File(filepath);
+        File tempo = new File(temp);
+        PrintWriter out = new PrintWriter(new FileWriter(temp));
+        Files.lines(file.toPath())
+            .filter(line -> !line.contains(lineContent))
+            .forEach(out::println);
+        out.flush();
+        out.close();
+        tempo.renameTo(file);
+    }
+
+    /**
+     * add task to storage
+     * @param filepath
+     * @param textToAdd
+     * @throws IOException
+     */
+    public void writeToFile(String filepath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filepath);
+        fw.write(textToAdd);
+        fw.close(); 
+    }
+
+    /**
+     * list tasks from file
+     * @param filepath
+     * @throws FileNotFoundException
+     */
+    public void printFileContents(String filepath) throws FileNotFoundException {
+        File f = new File(filepath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
```

Comment:
Personally i would prefer naming my variable using a longer version, e.g. `scanner` instead of `s`. The variable is used quite shortly after this so it is still okay, but if after about 20 lines of code, i would probably forget what `s` stands for :)